src/edu/washington/escience/myria/expression/ConstantExpression.java

```diff
@@ -94,6 +95,14 @@ public ConstantExpression(final boolean value) {
   public ConstantExpression(final String value) {
     this(Type.STRING_TYPE, value);
   }
+  /**
+   * Construct Blob constant.
+   *
+   * @param value the value of this constant.
+   */
+  public ConstantExpression(final ByteBuffer value) {
```

Comment:
First, this function is never used, and second, the return value would be a summary string of the ByteBuffer. It would cause a problem for `getJavaString` since it can't be parsed as some value, and also for the `ConstantExpression` constructor since we're expecting a direct value like in a JSON string. I think an `UnsupportedOperationException` makes more sense if you don't really want to output all the bytes as string.